slaves/blender/blender.ino

```diff
@@ -37,6 +24,57 @@ void setup() {
     Wire.onRequest(sendData);
 }
 
+void loop() {
+    // Debug
+    // digitalWrite(ADD_D24,LOW);
+    // delay(1000);
+    // digitalWrite(ADD_D24,HIGH);
+    // delay(1000);
+
+    // Output state machine
+
+    // Update sensor values
+
+}
+
+// callback for received data
+void receiveData(int byteCount) {
+    if (Wire.available()) {
+        COMM_SELECTOR selector = (COMM_SELECTOR)Wire.read();
+        if (Wire.available()) {
+            uint8_t data = Wire.read();
+            switch (selector) {
+                case BLEND1:
```

Comment:
I don't think this will work... you're reading one byte at a time meaning you're just casting your selector byte to a `BLENDER`

Reply:
https://www.arduino.cc/en/Reference/WireRead